explorer/types.go

```diff
@@ -368,3 +370,122 @@ type HostMetrics struct {
 	Settings   rhpv2.HostSettings   `json:"settings"`
 	PriceTable rhpv3.HostPriceTable `json:"priceTable"`
 }
+
+// HostSortDir represents the sorting direction for host filtering.
+type HostSortDir string
+
+const (
+	// HostSortAsc means sorting in ascending order.
+	HostSortAsc HostSortDir = "ASC"
```

Comment:
`nit` I'd use lower case values for sort dir here because they're passed as query string params